tools/bin/cargo-elf2sgxs.rs

```diff
@@ -71,72 +71,80 @@ fn real_main() -> Result<()> {
     }
     let package = package_root.package().unwrap();
 
-    // Build target path.
+    // Build target directory.
     let mut target_path = package_root.target_path();
     target_path.push(TARGET_TRIPLE);
     if matches.is_present("release") {
         target_path.push("release");
     } else {
         target_path.push("debug");
     }
-    // TODO: Discover binary name (what cargo does).
-    target_path.push(&package.name);
-
-    // Populate elf2sgxs arguments.
-    let config = &package.metadata.fortanix_sgx;
-    let heap_size = config.heap_size.unwrap_or(DEFAULT_HEAP_SIZE).to_string();
-    let ssaframesize = config
-        .ssaframesize
-        .unwrap_or(DEFAULT_SSAFRAMESIZE)
-        .to_string();
-    let stack_size = config.stack_size.unwrap_or(DEFAULT_STACK_SIZE).to_string();
-    let threads = config.threads.unwrap_or(DEFAULT_THREADS).to_string();
-    let debug = config.debug.unwrap_or(DEFAULT_DEBUG);
-
-    // Invoke ftx-elf2sgxs binary to perform the actual conversion.
-    println!(
-        "{} {} {} ({})",
-        Green.bold().paint(format!("{:>12}", "elf2sgxs")),
-        package.name,
-        package.version,
-        package_root.package_path().to_str().unwrap(),
-    );
-
-    // Compare source and target modification times and do not do anything
-    // if the target is newer.
-    let src_meta = fs::metadata(&target_path).context(format!(
-        "source file ({}) not found",
-        target_path.to_str().unwrap()
-    ))?;
-    if let Ok(target_meta) = fs::metadata(target_path.with_extension("sgxs")) {
-        let src_modified = src_meta.modified()?;
-        let target_modified = target_meta.modified()?;
-
-        if target_modified > src_modified {
-            println!(
-                "{} {}",
-                Green.bold().paint(format!("{:>12}", "elf2sgxs")),
-                White.dimmed().paint("(skipped due to newer target file)"),
-            );
-            return Ok(());
+    target_path.push("bin");
```

Comment:
Maybe call this `"<binary-name-placeholder>"` and add a comment because at first glance it looks like the binaries will go into a new `bin` directory (but this is in fact immediately pop-ed).